dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/support/fixed/FixedThreadPool.java

```diff
@@ -48,7 +48,7 @@ public Executor getExecutor(URL url) {
         int queues = url.getParameter(QUEUES_KEY, DEFAULT_QUEUES);
         return new ThreadPoolExecutor(threads, threads, 0, TimeUnit.MILLISECONDS,
                 queues == 0 ? new SynchronousQueue<Runnable>() :
-                        (queues < 0 ? new LinkedBlockingQueue<Runnable>()

```

Comment:
I think the key point here is not the type of Queue but is how different values of `queues` is treated:
* 0, SynchronousQueue
* <0, LinkedBlockingQueue
* >0, limited LinkedBlockingQueue